certified-connectors/Mural/apiDefinition.swagger.json

```diff
@@ -1,1074 +1,1394 @@
-        "description": "Get Murals",
-        "operationId": "GetMurals",
-        "x-ms-visibility": "internal",
-        "parameters": [
-          {
-            "name": "roomId",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "x-ms-url-encoding": "single",
-            "x-ms-dynamic-values": {
-              "operationId": "GetRoomsOperation",
-              "value-path": "id",
-              "value-collection": "value",
-              "value-title": "name",
-              "parameters": {
-                "workspaceId": {
-                  "parameter": "workspaceId"
-                }
-              }
-            },
-            "x-ms-dynamic-list": {
-              "operationId": "GetRoomsOperation",
-              "itemsPath": "value",
-              "itemValuePath": "id",
-              "itemTitlePath": "name",
-              "parameters": {
-                "workspaceId": {
-                  "parameterReference": "workspaceId"
-                }
-              }
-            }
-          },
-          {
-            "name": "workspaceId",
-            "in": "query",
-            "required": false,
-            "type": "string",
-            "description": "Workspace",
-            "x-ms-summary": "Workspace",
-            "x-ms-dynamic-values": {
-              "operationId": "GetWorkspaceOperation",
-              "value-path": "id",
-              "value-collection": "value",
-              "value-title": "name"
-            }
-          }
-        ]
-      }
-    }
-  },
-  "definitions": {},
-  "parameters": {},
-  "responses": {},
-  "securityDefinitions": {
-    "oauth2_auth": {
-      "type": "oauth2",
-      "flow": "accessCode",
-      "authorizationUrl": "https://app.mural.co/api/public/v1/authorization/oauth2/",
-      "tokenUrl": "https://app.mural.co/api/public/v1/authorization/oauth2/token",
-      "scopes": {
-        "rooms:read users:read workspaces:read murals:read murals:write identity:read templates:read": "rooms:read users:read workspaces:read murals:read murals:write identity:read templates:read"
-      }
-    }
-  },
-  "security": [
-    {
-      "oauth2_auth": [
-        "rooms:read users:read workspaces:read murals:read murals:write identity:read templates:read"
-      ]
-    }
-  ],
-  "tags": []
-}
\ No newline at end of file
+	"swagger": "2.0",
+	"info": {
+		"title": "MURAL",
+		"description": "Connect MURAL with your favorite apps, so you can automate your work and have more time for what matters most—no code required.",
+		"version": "1.0",
+		"contact": {
+			"name": "MURAL Support",
+			"url": "https://www.mural.co/support",
+			"email": "support@mural.co"
+		}
+	},
+	"host": "app.mural.co",
+	"basePath": "/",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [],
+	"paths": {
+		"/api/public/v1/murals": {
+			"post": {
+				"responses": {
+					"201": {
+						"description": "default",
```

Comment:
@rmiller-mural Please change to a proper description. for example you can use "OK", "Success", "Bad Request", ...etc.
please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).
Please fix all responses in your connector swagger file.